lib/jsonapi-response.js

```diff
@@ -0,0 +1,62 @@
+const attributesMap = require('../fixtures/attributes.js');
+const relationshipsMap = require('../fixtures/relationships.js');
+
+module.exports = (data) => {
+  const response = {};
+
+  response.data = {};
+  response.data.type = data._type;
+  response.data.id = data._id;
+  response.data.attributes = getAttributes(data._source);
+
+  response.relationships = getRelationships(data._source);
+
+  response.links = {};
+  response.links.self = 'http://www.sciencemuseum.org.uk/' + data._type + '/' + data._id;
```

Comment:
Config now has a [`rootUrl`](https://github.com/TheScienceMuseum/collectionsonline/blob/master/.corc.template#L3) item which you should be able to pass through here